src/datasources/mono/mono_usage.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const MonoDataSource = require('./mono.js')
+const monoDS = new MonoDataSource(process.env.MONO_TOKEN);
+(async () => {
+    try {
+        const transactions = await monoDS.getTransactions(undefined, new Date(2020, 10, 15), new Date(2020, 10, 26));
```

Comment:
Leave some comment that it's desirable behavior or something, it's a bit confusing that you pass undefined here. Maybe, add some successfull example too.

Reply:
ok